save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ExecutionService.kt

```diff
@@ -287,6 +289,26 @@ class ExecutionService(
         return savedExecution
     }
 
+    /**
+     * Unlink provided [File] from all [Execution]s
+     *
+     * @param file
+     */
+    @Transactional
+    fun unlinkFileFromAllExecution(file: File) {
```

Comment:
It's not actually an `unlink`, isn't it? We don't remove corresponding records from `lnkExecutionFileRepository`
So, maybe call it like `markeExecutionsCorrespondingToFileAsObsolete`?

Reply:
actually, it should delete `lnkExecutionFileRepository`.
Fixed the method